src/fuel-vm/instruction-set.md

```diff
@@ -1796,33 +1796,35 @@ Panic if:
 - `$rB + 32` overflows or `> VM_MAX_RAM`
 - Contract with ID `MEM[$rB, 32]` is not in `tx.inputs`
 
-### `LDC`: Load code from an external contract or blob
+### `LDC`: Load code from an external contract, blob or memory
 
 |             |                                                                                                                                                   |
 |-------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
-| Description | Copy `$rC` bytes of code at offset `$rB` from object with 32 byte id starting at `$rA` into memory starting at `$ssp`. Object type is in `imm`.   |
-| Operation   | `id = mem[$rA,32]; code = match imm { 0 => contract_code($id), 1 => blob_payload($id) }; MEM[$ssp, $rC] = code[$rB, $rC];`                        |
+| Description | Copy `$rC` bytes of code at offset `$rB` from object identified with `$rA` into memory starting at `$ssp`. Object type is in `imm`.               |
```

Comment:
nit: it would be nice to explain exactly what it is meant by "identified". E.g. something "with hash equivalent to the content of `$ra`.

Reply:
But it's not always a hash, which is why I use an abstract wording here. The table below explains the types.